test/functional/feature_walletestimatefee.py

```diff
@@ -0,0 +1,269 @@
+#!/usr/bin/env python3
```

Comment:
I know you did this originally, but is it really not worthwhile to have "test/functional/feature_fee_estimation.py" take on this testing? I feel like this is a lot of duplicated code. Thoughts?